### PR TITLE
Kathirsvn/readme update

### LINKED ISSUE #13: Review & Cleanup Supported operations in README

### PR DESCRIPTION
**Pull request by kathirsvn**
**What this PR does**:
README is updated with the latest updates to the driver

**Which issue(s) this PR fixes**:
Fixes #13 

**Checklist**
- [X] Changes manually tested
- [X] Automated Tests added/updated
- [X] Documentation added/updated
- [X] CLA Signed: [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,85 +131,80 @@ https://github.com/stargate/stargate-mongoose-sample-apps
...
+| find                        | `Model.find(filter, projection, options)`  options - limit, pageState, skip (skip works only with sorting)                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| findOne                     | `Model.findOne(filter)` findOne doesn't take any options                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
```
Comment (vkarpov15) [inline README.md]: findOne() does take options. `sort` for example.

File: README.md [hunk 2]
```diff
@@ -131,85 +131,80 @@ https://github.com/stargate/stargate-mongoose-sample-apps
...
+| findOneAndDelete            | `Model.findOneAndDelete(filter)`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
```
Comment (vkarpov15) [inline README.md]: Add options params to `findOneAndX()`

**Review by jeffreyscarpenter — APPROVED** [review]

**Review by vkarpov15 — CHANGES_REQUESTED** [review]
A couple of minor details, but otherwise LGTM

File: README.md (hunk 1 above)
Comment (kathirsvn) [reply README.md]: Fixed in [9d2336b](https://github.com/stargate/stargate-mongoose/pull/92/commits/9d2336b652f8fb9f8cf7a67dee4d49c61ba38244)

File: README.md (hunk 2 above)
Comment (kathirsvn) [reply README.md]: Fixed in [9d2336b](https://github.com/stargate/stargate-mongoose/pull/92/commits/9d2336b652f8fb9f8cf7a67dee4d49c61ba38244)

**Review by vkarpov15 — APPROVED** [review]
LGTM :+1:

**Pull request merged**